.github/scripts/validate.js

```diff
@@ -0,0 +1,209 @@
+const FIELD_MAPPINGS = {
+  Platform: 'What platforms are you having the problem on?',
+  Version: 'Version',
+  SystemVersion: 'System Version',
+  DeviceType: 'On what device are you experiencing the issue?',
+  Architecture: 'Architecture',
+  Description: 'What happened?',
+  ReproductionLink: 'Reproduction Link',
+  Reproduction: 'Reproduction',
+};
+
+const PLATFORM_LABELS = {
+  iOS: 'Platform: iOS',
+  visionOS: 'Platform: iOS',
+  'Apple tvOS': 'Platform: iOS',
+  Android: 'Platform: Android',
+  'Android TV': 'Platform: Android',
+  Windows: 'Platform: Windows',
+  web: 'Platform: Web',
+};
+
+const BOT_LABELS = [
+  'Missing Info',
+  'Repro Provided',
+  'Missing Repro',
+  'Waiting for Review',
+  ...Object.values(PLATFORM_LABELS),
+];
+
+const MESSAGE = {
+  FEATURE_REQUEST: `Thank you for your feature request. We will review it and get back to you if we need more information.`,
+  BUG_REPORT: `Thank you for your bug report. We will review it and get back to you if we need more information.`,
+  MISSING_INFO: (missingFields) => {
+    return `Thank you for your issue report. Please note that the following information is missing or incomplete:\n\n${missingFields
+      .map((field) => `- ${field.replace('missing-', '')}`)
+      .join(
+        '\n',
+      )}\n\nPlease update your issue with this information to help us address it more effectively. 
+      \n > Note: issues without complete information have a lower priority`;
+  },
+};
+
+const getFieldValue = (body, field) => {
+  if (!FIELD_MAPPINGS[field]) {
+    console.warn('Field not supported:', field);
+    return '';
+  }
+
+  const sections = body.split('###');
+  const section = sections.find((section) => {
+    // Find the section that contains the field
+    // For Reproduction, we need to make sure that we don't match Reproduction Link
+    if (field === 'Reproduction') {
+      return (
+        section.includes(FIELD_MAPPINGS[field]) &&
+        !section.includes('Reproduction Link')
+      );
+    }
+
+    return section.includes(FIELD_MAPPINGS[field]);
+  });
+
+  return section ? section.replace(FIELD_MAPPINGS[field], '').trim() : '';
+};
+
+const validateBugReport = (body, labels) => {
+  const selectedPlatforms = getFieldValue(body, 'Platform')
+    .split(',')
+    .map((p) => p.trim());
+
+  if (selectedPlatforms.length === 0) {
+    labels.add('missing-platform');
+  } else {
+    selectedPlatforms.forEach((platform) => {
+      const label = PLATFORM_LABELS[platform];
+      if (label) {
+        labels.add(label);
+      } else {
+        console.warn('Platform not supported', platform);
+      }
+    });
+  }
+
+  const fields = [
+    {
+      name: 'Version',
+      invalidValue: 'Put the exact version from your package.json',
+    },
+    {
+      name: 'SystemVersion',
+      invalidValue:
+        'What version of the system is using device that you are experiencing the issue?',
+    },
+    {name: 'DeviceType'},
```

Comment:
What is your opinion on adding invalidValue for `DeviceType` and `Architecture`?